src/main/kotlin/mate/academy/Main.kt

```diff
@@ -1,7 +1,18 @@
 package mate.academy
 
 fun greetUsers() {
-    // implement function
+    fun greet(name: String) {
```

Comment:
Inner function in this case is overengineering a little as this is just wrapper on println(). You may call it directly